sdk/compiler/damlc/lib/DA/Cli/Damlc/Command/MultiIde.hs

```diff
@@ -226,14 +226,14 @@ disableIdeDiagnosticMessages ideData home =
     )
     <$> ((home </> "daml.yaml") : Set.toList (ideDataOpenFiles ideData))
 
-runSubProc :: FilePath -> IO (Process Handle Handle Handle)
-runSubProc home = do
+runSubProc :: MultiIdeState -> FilePath -> IO (Process Handle Handle Handle)
+runSubProc miState home = do
   assistantPath <- getEnv "DAML_ASSISTANT"
   -- Need to remove some variables so the sub-assistant will pick them up from the working dir/daml.yaml
   assistantEnv <- filter (flip notElem ["DAML_PROJECT", "DAML_SDK_VERSION", "DAML_SDK"] . fst) <$> getEnvironment
 
   startProcess $
-    proc assistantPath ["ide"] &
+    proc assistantPath ("ide" : subIdeArgs miState) &
```

Comment:
If we're passing unrecognized arguments from multiide down to sub ides, those subides should be able to cope with unrecognized arguments too - can we add the `forwardOptions` modifier to `cmdIde` as well? Maybe emitting a warning from IDE so that unrecognized arguments can be debugged.

Reply:
I believe the ide is already running with a lax parser, but I'll check

Reply:
It's not, they do error but the user is informed elegantly. I feel like it's probably valuable to tell the user when these don't work, rather than silently ignore them. For anything package specific, they should live in the `daml.yaml` build-options